tcc/src/main/java/io/seata/rm/tcc/TCCResourceManager.java

```diff
@@ -38,6 +38,7 @@
  * TCC resource manager
  *
  * @author zhangsen
+ * @author Yujianfei
```

Comment:
author information should not be added without refactoring

Reply:
The author on this class should not be removed, because this class is one of the core changes of this PR.